Doc : Add mention that invoices are sent to primary and secondary mail (#560)
CleverCloud/documentation: content/doc/billing/payments-invoicing.md

### About invoices issuance and notifications

As explained in the [Monthly Invoice documentation]({{< ref "doc/billing/unified-invoices.md#monthly-invoice" >}}), Clever Cloud issues an invoice at the beginning of every month.  

> The invoice is emailed to you. It's sent to both the primary and secondary mail of every Admin or Accountant in the organization.  

Reviewer: ```suggestion
It's sent to both primary and secondary email address of any Admin or Accountant within the organization.  
```
